components/form-builder/layout/TestDataDelivery.tsx

```diff
@@ -57,6 +57,8 @@ export const TestDataDelivery = () => {
     <div>
       <div className="mb-8 bg-blue-200 p-5">
         {t("submittedResponsesText", { email })}{" "}
+        {/* TODO this should probably be a button - for a future refactor */}
```

Comment:
Thinking about this now, flagging TODOs in the code may not be the best way. But I'll leave this here for now - curious what others thing?